lib/src/main/kotlin/ca/aqtech/mctreesearch4j/ActionNode.kt

```diff
@@ -26,14 +43,17 @@ class ActionNode<StateType, ActionType> (
 
     private var children = mutableListOf<ActionNode<StateType, ActionType>>()
 
+    // Inherited doc comments
```

Comment:
@JunTaoLuo should we remove these, are they necessary?

Reply:
I added these comments to remind ourselves that even though we didn't write explicit doc comments for these overrides, they will inherit the doc comments from the base class.